apps/blazor-test-app/Blazor-Test-App.csproj

```diff
@@ -25,6 +25,7 @@
     <PackageReference Include="Microsoft.Fast.Components.FluentUI" Version="1.5.3" />
     <PackageReference Include="Microsoft.QualityTools.Testing.Fakes" Version="16.11.230815" />
     <PackageReference Include="Microsoft.TeamsFx" Version="1.2.*" />
+    <PackageReference Include="System.Drawing.Common" Version="4.7.2" />
```

Comment:
Looking at here, it says System.drawing.Common only supported on Windows?
https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-And starting with .net6, we might start to build warning when running on non-windows machine.

so it feels we could evaluate a few options: is that possible that we find a replacement for System.Drawing.Common to be used in test app? and if not, maybe we could find a way to run this blazor test app on window agent from mobile side?